core/file_api.php

```diff
@@ -69,8 +69,40 @@ function file_attach_files( $p_bug_id, $p_files, $p_bugnote_id = 0 ) {
 	}
 
 	$t_file_infos = array();
+	$t_zip_attachments = config_get( 'zip_attachments' );
```

Comment:
@dregad I think we should not allow this setting to be a project / user specific one, so config_get_global should be used. Do you agree?

Reply:
Can of course change it but depends on your first comment blacklist versus whitelist)  if we still need this setting